admin_guide/limit_pod_access_egress.adoc

```diff
@@ -0,0 +1,104 @@
+[[admin-guide-limit-pod-access-egress]]
+= Limit Pod Access with Egress Firewall
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+As an {product-title} cluster administrator, you can use egress policy to limit
+the addresses that some or all pods can access from within the cluster, so that:
+
+- A pod can only talk to internal hosts, and cannot initiate connections to the
+public Internet.
++
+Or, 
+- A pod can only talk to the public Internet, and cannot initiate connections to
+internal hosts (outside the cluster).
++
+Or,
+- A pod cannot reach specified internal subnets/hosts that it should have no
+reason to contact.
+
+For example, you can configure projects with different egress policies, allowing
+`<project A>` access to a specified IP range, but denying the same access to
+`<project B>`. 
```

Comment:
I don't think the "Or" things work too well here, plus, they don't fit with other bullet points we've donw in the rest of the docs. I think they could go, and the "so that" bit at the top could be more like "for three reasons". Let me know if that doesn't make sense.